apps/agent/README.md

```diff
@@ -0,0 +1,128 @@
+# Agent
+
+
+
+## 1. Mechanism
```

Comment:
Perhaps "mechanism" will make more sense once we have this section written, but we may find a better wording :).